### PR TITLE
Enable test for PostgreSQL-based metastore on CI

### PR DESCRIPTION
**Pull request by mosuka**
### Description
Enable test for PostgreSQL-based metastore on CI.
Also, changed the feature name from postgresql to postgres.

### CODE REVIEW

File: quickwit-metastore/docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
-version: "3.9" 
+version: "3.9"
```
Comment (guilload) [inline quickwit-metastore/docker-compose.yml:1]: I recently committed a docker-compose file at the root of the project; we can work off that one and delete this one.

File: quickwit-metastore/.env [hunk 2]
```diff
@@ -1,4 +1,5 @@
 POSTGRES_USER=quickwit-dev
 POSTGRES_PASSWORD=quickwit-dev
-DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost/quickwit-dev
-TEST_DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost/quickwit-dev
+POSTGRES_DB=quickwit-metastore
+DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost:5432/quickwit-metastore
```
Comment (guilload) [inline quickwit-metastore/.env:4]:
```suggestion
DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost:5432/quickwit-metastore-dev
```

File: quickwit-metastore/.env [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+TEST_DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost:5432/quickwit-metastore
```
Comment (guilload) [inline quickwit-metastore/.env:5]: Why do we need both `DATABASE_URL` and `TEST_DATABASE_URL` if they are the same? I'd rather keep `TEST_DATABASE_URL,` which has a more explicit name.

File: quickwit-metastore/README.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ $ docker-compose up --force-recreate -d
 Once PostgreSQL is up and running, you can run tests including PostgresqlMetastore with the following command.
 
 ```
-$ cargo test --features=postgresql
+$ cargo test --features=postgres
```
Comment (guilload) [inline quickwit-metastore/README.md:72]: You'll have to modify this README to refer to the "main" docker-compose: `make docker-compose-up DOCKER_SERVICES=postgres`

**Review by guilload — CHANGES_REQUESTED** [review]

File: quickwit-metastore/.env (hunk 3 above)
Comment (mosuka) [reply quickwit-metastore/.env:5]: Yes, we don't need the `DATABASE_URL` here. I'll delete it.

**Comment by mosuka** [comment]
@guilload 
Thank you for the review.
I've updated the branch, so please take another look.

File: quickwit-metastore/.env (hunk 3 above)
Comment (PSeitz) [reply quickwit-metastore/.env]: If this is only for tests then the file location is not ideal

Comment (mosuka) [reply quickwit-metastore/.env]: @PSeitz 
You're right, it's better to put it in the same place as docker-compose.yml.

**Pull request merged**